include/unifex/detail/debug_async_scope.hpp

```diff
@@ -0,0 +1,209 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License Version 2.0 with LLVM Exceptions
+ * (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ *   https://llvm.org/LICENSE.txt
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#pragma once
+
+#include <unifex/detail/intrusive_list.hpp>
+
+#include <cxxabi.h>
+#include <mutex>
+#include <typeinfo>
+
+#include <unifex/detail/prologue.hpp>
+
+namespace unifex::detail {
+namespace _debug_async_scope {
+
+struct op_base {
+  op_base(const std::type_info& concreteType) noexcept
```

Comment:
This should probably be explicit.
```suggestion
  explicit op_base(const std::type_info& concreteType) noexcept
```